linux_os/guide/services/obsolete/tftp/tftp_uses_secure_mode_systemd/rule.yml

```diff
@@ -0,0 +1,38 @@
+documentation_complete: true
+
+title: 'Ensure tftp Daemon Uses Secure Mode Using systemd'
+
+description: |-
+    If running the Trivial File Transfer Protocol (TFTP) service is necessary,
+    it should be configured to change its root directory at startup. To do so,
+    find the path for the <tt>tftp</tt> systemd service:
+    <pre>$ sudo systemctl show tftp | grep ExecStart=
+    ExecStart={ path=/usr/sbin/in.tftpd ; argv[]=/usr/sbin/in.tftpd -s /var/lib/tftpboot ; ignore_errors=no ; start_time=[n/a] ; stop_time=[n/a] ; pid=0 ; code=(null) ; status=0/0 }e
+    </pre>
+
+    and ensure the <tt>ExecStart</tt> line on that file includes the <tt>-s</tt> option with a subdirectory:
+    <pre>ExecStart=/usr/sbin/in.tftpd -s {{{ xccdf_value("var_tftpd_secure_directory") }}}</pre>
+
+rationale: |-
+    Using the <tt>-s</tt> option causes the TFTP service to only serve files from the
+    given directory. Serving files from an intentionally-specified directory
+    reduces the risk of sharing files which should remain private.
+
+severity: medium
+
+ocil: |-
+    Use <tt>sudo systemctl edit tftp</tt> to adjust the <tt>ExecStart</tt> to
+    be <tt>/usr/sbin/in.tftpd -s {{{ xccdf_value("var_tftpd_secure_directory") }}}</tt>
+
+
+ocil_clause: 'the ExecStart property of tftp does not contain correctly set -s flag'
+
+platform: package[tftp-server]
+
+warnings:
+    - general: |-
+        A remedation is not currently available due limits of the checking engine.
```

Comment:
The checking engine isn't a reason of a missing remediation. You can have remediations that would be executed if they run separately, Ansible Playbook or a Bash remediation.

I suggest rewording this and explaining why there is no OVAL check.